.github/workflows/ci.yml

```
@@ -65,34 +67,24 @@ jobs:
       - name: Install dependencies
         uses: ramsey/composer-install@v2
 
-      - name: Load php-cs-fixer cache
-        id: php-cs-fixer-cache
-        uses: actions/cache/restore@v3
+      - name: Cache PHP-CS-Fixer
+        uses: actions/cache@v3
         with:
-          path: .php-cs-fixer.cache
-          key: php-cs-fixer-${{ github.ref_name }}
-          restore-keys: |
-            php-cs-fixer-
+          path: ${{ env.PHP_CS_FIXER_CACHE_DIR }}
+          key: php-cs-fixer-${{ runner.os }}-${{ steps.setup_php.outputs.php-version }}
```

We're only running 1 check in the up-to-date PHP version, so the php-version in the key is superfluous. Also, we're only running one OS, so the key can be very simple, right?
```suggestion
          key: php-cs-fixer
```

---

These two values are preparations for OS and PHP version change in the future.
I am 100% sure OS and PHP will change.

---

I hope you don't plan to purge caches manually!

---

Even if the PHP version would change, we can still use the previous cached PHP version cache and update the cache afterwards. I live by YAGNI principles. ;) If these changes, we'll tackle this issue at that time.

---

I have _zero risk taking_ sickness.